gtsam/geometry/Similarity3.cpp

```diff
@@ -174,18 +174,19 @@ Similarity3 Similarity3::Align(const vector<Pose3Pair> &abPosePairs) {
 
   // calculate rotation
   vector<Rot3> rotations;
-  PointPairs abPointPairs;
+  Point3Pairs abPointPairs;
   rotations.reserve(n);
   abPointPairs.reserve(n);
-  Pose3 wTa, wTb;
+  Pose3 aTi, bTi;
```

Comment:
Please add a comment on what frame `i` is?

Reply:
done, thanks